### DISCUSSION TITLE
Personal Playlists

### DISCUSSION THREAD

**Discussion by jrpetersjr** [Taxel/PlexTraktSync#1625] [Q&A] [answered]
Hi,

I'm looking to possibly start using this, but I don't see anything about syncing personal list...I only see liked lists. Does this sync your personal lists as well? I've got 30 different lists that I would like to sync.

**Comment by simonc56** [comment, marked as answer]
Yes you can sync your personnal lists.
You just have to like them and they will be synced as liked lists.
